### PR TITLE
Start PR template w/ a11y reqs

### PR DESCRIPTION
**Pull request by melanierichards**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,12 @@ too sentimental). Some others are new. In any case, we are working on adding mor
 1. Fork this project and [set up a remote](https://help.github.com/articles/configuring-a-remote-for-a-fork/) to file pull requests
 against later.
 2. Create a feature branch for your new demo off of the master branch.
-3. Before creating a pull request, make sure your feature branch is up to date with the latest changes to MicrosoftEdge/Demos/master (the
+3. Before starting any work, and preferrably before designing the experience of your demo, review the [accessibility requirements](.github/ACCESSIBILITY_REQS.md) that your demo must meet.
```
Comment (molant) [inline README.md]: Maybe say something about reaching out to @melanierichards ?

File: README.md [hunk 2]
```diff
@@ -14,11 +14,12 @@ too sentimental). Some others are new. In any case, we are working on adding mor
...
+4. Before creating a pull request, make sure your feature branch is up to date with the latest changes to MicrosoftEdge/Demos/master (the
 remote you set up).
-4. Create a pull request against MicrosoftEdge/Demos/master with the changes from your branch. Title with the name of your demo or fixes.
-Mention @molant and optionally @melanierichards (for front-end/design review) in the comments so we're aware of your PR.
-5. Push any changes based on feedback to your feature branch. This will update the PR with the most recent changes.
+5. Create a pull request against MicrosoftEdge/Demos/master with the changes from your branch. Title with the name of your demo or fixes.
+Request a review from @molant and @melanierichards. If your PR contains a new demo, it must be submitted at least one week in advance of the intended publication.
+6. Push any changes based on feedback to your feature branch. This will update the PR with the most recent changes.
 
 ### What code should I use?
```
Comment (molant) [inline README.md:29]: Should we just tell them to use plain HTML now that we have support for that?

**Review by molant — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (melanierichards) [reply README.md:29]: We decided offline to create a very basic demo template within this repo, but will do it in another PR to keep concerns separate.

File: .github/ACCESSIBILITY_REQS.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+## Audio and video
+* If audio plays for more than 3 seconds, users have a way to pause or stop that audio or adjust its volume
+* Content is not dependent on audio/sensory info alone to convey information, indicate an action, prompt a response, or distinguish a visual element
+* Pre-recorded media (anything not captured in real time, such as webcam feeds) have captions, audio descriptions, and a transcript
+* Media capture in real time have captions
+
+## Text
+* Text can be resized without assistive technology to +200%, without losing content or functionality. Exceptions: captions and images of text
+* Text-in-image is not relied upon to convey information. Exceptions: when the info is dependent upon the image format, such as logos or typography samples
+* The page language (English, for ex.) is set in metadata, and any exceptions to the main documentation language are noted using a `lang` attribute
```
Comment (dstorey) [inline .github/ACCESSIBILITY_REQS.md]: can you change ex. to example or e.g. (not sure if this is a MSFTism or American-isn but ex. can also mean exclusion and other things)

File: .github/ACCESSIBILITY_REQS.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Color
+
+* Content is not dependent on color alone to convey information, indicate an action, prompt a response, or distinguish a visual element
+* All text needs a 4.5:1 color ratio against its background. [Here's a place where you can test color combos](http://leaverou.github.io/contrast-ratio/) Exceptions: logos, "incidental" text (decorative text, text in an inactive element)
```
Comment (dstorey) [inline .github/ACCESSIBILITY_REQS.md]: probably worth just telling person to scan the page, as it is faster than testing by hand,

File: .github/ACCESSIBILITY_REQS.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## Context
+* All non-text content has a text alternative
+* The value of the page title is informative and specific to the current page
+* Link text, wherever possible, informs the user of where the link goes without having to rely on other items for context
```
Comment (dstorey) [inline .github/ACCESSIBILITY_REQS.md]: probably worth mentioning here that "read more" style links is the common issue here.

File: .github/ACCESSIBILITY_REQS.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+# Accessibility
+
+I can verify that the changes made by my PR meet all the following accessibility concerns:
+
```
Comment (dstorey) [inline .github/ACCESSIBILITY_REQS.md:4]: most of these are WCAG items, so where possible I'd link to the understanding WCAG document I link to for each rule on our a11y review, as it gives more context and gives techniques to pass and examples of passing and failing.

**Review by dstorey — COMMENTED** [review]
Few small points.

**Comment by melanierichards** [comment]
@dstorey ready to review again!

**Review by dstorey — APPROVED** [review]

**Pull request merged**